surfsense_web/app/dashboard/page.tsx

```
@@ -211,6 +211,7 @@ const DashboardPage = () => {
 
           <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6">
             {searchSpaces && searchSpaces.map((space) => (
+              <Link href={`/dashboard/${space.id}/documents`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)